packages/venia-concept/src/components/Gallery/items.js

```diff
@@ -55,12 +55,23 @@ class GalleryItems extends Component {
         this.setState(initState);
     }
 
+    get placeholders() {
+        const { pageSize } = this.props;
+        const sizedEmptyData = Array.from({ length: pageSize }).fill(null);
```

Comment:
This is good. Since `Array.from()` takes a mapping function as a second argument, and doesn't create an intermediate array, why not combine them below?